### PR TITLE
OD-251 [Feature] Filter list updates

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
https://weboo.atlassian.net/jira/software/projects/OD/boards/9?selectedIssue=OD-251
## Description
Update filter list fields
## Screenshots/screencasts
https://screencast-o-matic.com/watch/cYlVlANQxj
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Review by tonytlwu — CHANGES_REQUESTED** [review]
Branch out of date

**Comment by AndrRyaz** [comment]
@tonytlwu updated

**Comment by tonytlwu** [comment]
Issue description mentioned OD-267 and that's incorrect.

The 3rd acceptance criterion and scenario 4 described that the field label should say `(Field)` when none is selected. The sreencast showed that when none is selected, it doesn't say `(Field)`.

**Comment by AndrRyaz** [comment]
@tonytlwu fixed

**Comment by AndrRyaz** [comment]
@tonytlwu updated 
https://screencast-o-matic.com/watch/cYlVD7NXXS

File: templates/interface/filter-panels.interface.hbs [hunk 1]
```diff
@@ -6,10 +6,6 @@
       </div>
       <span class="panel-title-text">
         <span class="column">{{columnLabel}}</span>
-        <span> - </span> 
-        <span class="logic">{{logicLabel}}</span>
```
Comment (tonytlwu) [inline templates/interface/filter-panels.interface.hbs:10]: The code that is used to populate this isn't needed any more. See interface.js.

**Review by tonytlwu — CHANGES_REQUESTED** [review]

**Comment by AndrRyaz** [comment]
@tonytlwu updated and removed the code that we don't need anymore

**Review by tonytlwu — CHANGES_REQUESTED** [review]
https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/4dc28a8cf2ace39f8d8e0400d953430ece07087b/js/interface.js#L248 isn't needed either. It doesn't look like the review of unused code is thorough enough.

**Comment by AndrRyaz** [comment]
@tonytlwu fixed and remove unused code, next time i will be more attentive

**Comment by AndrRyaz** [comment]
@tonytlwu fixed and remove unused code, next time i will be more attentive

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Low
Affected: Studio users configuring LFD prefilters
Verify: Filters can be correctly set up and loaded

**Pull request merged**